plugins/search-react/api-report.md

```diff
@@ -269,13 +271,22 @@ export type SearchResultGroupLayoutProps<FilterOption> = ListProps & {
   link?: ReactNode;
   linkProps?: Partial<LinkProps>;
   filterOptions?: FilterOption[];
-  renderFilterOption?: (filterOption: FilterOption) => JSX.Element;
+  renderFilterOption?: (
+    value: FilterOption,
+    index: number,
+    array: FilterOption[],
+  ) => JSX.Element | null;
   filterFields?: string[];
   renderFilterField?: (key: string) => JSX.Element | null;
   resultItems?: SearchResult_2[];
-  renderResultItem?: (resultItem: SearchResult_2) => JSX.Element;
+  renderResultItem?: (
+    value: SearchResult_2,
+    index: number,
+    array: SearchResult_2[],
```

Comment:
It looks like this (and the `renderResultItem` equivalent above) could have been an unintentional API addition as a result of passing the `renderResultItem` handler directly to a `.map`?  ...But perhaps having the `index` and the full array could be useful.

Just want to confirm: do you want to add these two arguments to the API of the `renderResultItem` (and `renderFilterOption`) handler(s)?

Reply:
Aaaah, makes sense.  That's great.